shared_model/interfaces/queries/impl/account_asset_pagination_meta.cpp

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "interfaces/queries/account_asset_pagination_meta.hpp"
+
+using namespace shared_model::interface;
+
+bool AccountAssetPaginationMeta::operator==(const ModelType &rhs) const {
+  return pageSize() == rhs.pageSize() and firstAssetId() == rhs.firstAssetId();
+}
+
+std::string AccountAssetPaginationMeta::toString() const {
+  return detail::PrettyStringBuilder()
+      .init("AccountAssetPaginationMeta")
+      .append("page_size", std::to_string(pageSize()))
+      .append("first_asset_id", firstAssetId().value_or("(none)"))
```

Comment:
I suggest to check if it is possible to reuse `operator<<` from `variant_io.hpp`.

Reply:
Decided to refactor in another task.